### DISCUSSION TITLE
error while execuritung the script

### DISCUSSION THREAD

**Discussion by aee74** [Staubgeborener/klipper-backup#102] [Q&A] [answered]
When I execute the script I get these errors.

The .env file has been updated whith a newly generated token, my own username and my repository

`github_token="#######" the token is formatted as ghp_*** 
github_username=aee74
github_repository=temporary1.0
branch_name=main
commit_username=""
commit_email=""

backupPaths=( \
"printer_data/config/*" \
)

exclude=( \
"*.swp" \
*.tmp" \
"printer-[0-9]*_[0-9]*.cfg" \
"*.bak" \
"*.bkp" \
"*.csv" \
*.zip" \
)
`

` ./script.sh
● Checking for installed dependencies Done!

Klipper-backup is up to date

fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
Branch: main in .env does not match the currently checked out branch of: .
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not in a git directory
fatal: not in a git directory
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
Skipping symbolic link: printer_data/config/KAMP
Skipping symbolic link: printer_data/config/ledcontrol.cfg
Skipping symbolic link: printer_data/config/timelapse.cfg
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: No remote configured to list refs from.
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
`

**Comment by Staubgeborener** [comment, marked as answer]
Try to delete your old backup and run the script again:

```
cd ~
rm -rf config_backup
```
